vizro-core/changelog.d/20240222_094020_maximilian_schulz_aggrid_separate_models.md

```diff
@@ -0,0 +1,48 @@
+<!--
```

Comment:
Once the docs are written so we can link to them then this should definitely go in the Highlights sections.

Reply:
I would add it in the docs PR - which would mark the official release I guess